alerta/app/database/mongo.py

```diff
@@ -49,8 +50,30 @@ def connect(self):
 
         mongo_uri = mongo_uri or app.config['MONGO_URI']  # use app config if no env var overrides
 
+        # Validate SSL options
+        def try_find(k):
+            try:
+                return (os.environ.get(k) or app.config[k])
+            except KeyError: pass
+
+        to_verify = ['MONGO_SSL_CERT', 'MONGO_SSL_CA']
+        ssl_opts = { k: try_find(k) for k in to_verify }
+        missing_ssl_opts = { k:v for k, v in ssl_opts.iteritems() if v is None }
```

Comment:
This seems overly complicated for what it does (and it doesn't work for python 3.x). How about something like this?
```
        ssl_args = {}
        ssl_cert = os.environ.get('MONGO_SSL_CERT', False) or app.config.get('MONGO_SSL_CERT', False)
        ssl_ca = os.environ.get('MONGO_SSL_CA', False) or app.config.get('MONGO_SSL_CA', False)

        if (ssl_cert and not ssl_ca) or (ssl_ca and not ssl_cert):
            LOG.error('Mongo SSL setup failed - must define MONGO_SSL_CERT and MONGO_SSL_CA')
            sys.exit(1)

        if ssl_cert and ssl_ca:
            ssl_args = {
                'ssl': True,
                'ssl_certfile': ssl_cert,
                'ssl_ca_certs': ssl_ca,
                'ssl_cert_reqs': ssl.CERT_REQUIRED
            }
```